web/cobrands/sass/_base.scss

```diff
@@ -1496,6 +1497,48 @@ html.js #map .noscript {
       background-image:url($image-sprite);
       background-position: flip(right, -341px) -3936px;
     }
+    &.expand {
+      padding: 0.6em 1em 0.5em 1em;
+      &:after {
+        content: "";
+        display: inline-block;
+        width: 16px;
+        height: 16px;
+        background-image: url(/cobrands/fixmystreet/images/expand.png);
+
+        @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 192dpi) {
+            background-image: url(/cobrands/fixmystreet/images/expand@2x.png);
+        }
+
+        @media all {
+            background-image: url(/cobrands/fixmystreet/images/expand.svg), none;
+        }
+        background-size: 16px 16px;
+        vertical-align: middle;
+        margin: flip(0 0 0 8px, 0 8px 0 0);
```

Comment:
This is fine, just to note that if you did only want to do the one margin as originally, you could write it as:
`margin-#{$left}: 8px;`.